test/test_dockerpy/README.md

```diff
@@ -2,29 +2,43 @@
 
 Python test suite to validate Podman endpoints using dockerpy library
 
+Validate :-
+
+Make sure podman binary exist under libpod/bin
+
+Validate Python 3.6+ installed using `python --version `
+
+Validate docker-py module exist by using `python list`. If missing install.
+
+
 Running tests
 =============
 To run the tests locally in your sandbox:
 
-#### Run the entire test
+cd into the libpod folder and run one of the following commands:-
+
+Note:- The test can be ran as root or rootless i.e sudo is optional.
+
+#### Run the entire test suite
 
 ```
-sudo PYTHONPATH=/usr/bin/python python -m dockerpy.images
+sudo PYTHONPATH=/usr/bin/python python -m unittest discover test/
 ```
 
 Passing the -v option to your test script will instruct unittest.main() to enable a higher level of verbosity, and produce detailed output:
 
 ```
-sudo PYTHONPATH=/usr/bin/python python -m unittest -v dockerpy.images
+sudo PYTHONPATH=/usr/bin/python python -m unittest discover -v test/
 ```
+
 #### Run a specific test class
 
 ```
-sudo PYTHONPATH=/usr/bin/python python -m unittest -v dockerpy.images.TestImages
+sudo PYTHONPATH=/usr/bin/python python -m unittest -v test.test_dockerpy.test_images
```

Comment:
Although I can't actually get it to work either way, I think it might be nicer to document that path as `test/test_dockerpy/test_images.py` -- that is, slashes instead of dots, and `.py` at the end. People use command completion, and it's much easier to type `test/test_do<TAB>/test_im<TAB>` than to spell out the entire thing by hand (and get it wrong, because tipyng is hard).